packages/x-charts/src/internals/plugins/featurePlugins/useChartPolarAxis/getAxisValue.ts

```diff
@@ -20,17 +20,25 @@ export function getAxisValue(axisConfig: PolarAxisDefaultized, pointerValue: num
       : Math.floor(angleGap / scale.step());
 
   if (dataIndex < 0 || dataIndex >= axisData!.length) {
-    return null;
+    return -1;
   }
-  if (reverse) {
-    const reverseIndex = axisData!.length - 1 - dataIndex;
-    return {
-      index: reverseIndex,
-      value: axisData![reverseIndex],
-    };
+
+  return reverse ? axisData!.length - 1 - dataIndex : dataIndex;
+}
+
+/**
+ * For a pointer coordinate, this function returns the value and dataIndex associated.
+ * Returns `null` if the coordinate is outside of values.
```

Comment:
It can also return `undefined`, right? Should we update the docs?

Reply:
It should not be possible. The computation of the `dataIndex` in the previouse selector is here to prevent that